pkg/controllers/user/monitoring/projectHandler.go

```diff
@@ -58,6 +58,54 @@ func (ph *projectHandler) sync(key string, project *mgmtv3.Project) (runtime.Obj
 	return cpy, err
 }
 
+// doRevertSync reverts all previous enabling Project Monitoring
+func (ph *projectHandler) doRevertSync(project *mgmtv3.Project, clusterName string) error {
+	if !mgmtv3.NamespaceBackedResource.IsTrue(project) && !mgmtv3.ProjectConditionInitialRolesPopulated.IsTrue(project) {
+		return nil
+	}
+
+	// drop metrics expressions
+	if !mgmtv3.ProjectConditionMetricExpressionDeployed.IsFalse(project) {
+		projectName := fmt.Sprintf("%s:%s", project.Spec.ClusterName, project.Name)
+
+		for _, graph := range preDefinedProjectGraph {
+			newObj := graph.DeepCopy()
+			newObj.Namespace = project.Name
+			newObj.Spec.ProjectName = projectName
+			if _, err := ph.app.cattleProjectGraphClient.Create(newObj); err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
why do we need to create cattleProjectGraphClient? Is it the component that's supposed to do the uninstall? If yes, when does the client itself get removed?